capa/main.py

```diff
@@ -698,9 +698,6 @@ def get_rules(
 
 
 def get_signatures(sigs_path):
-    if not os.path.exists(sigs_path):
-        raise IOError(f"signatures path {sigs_path} does not exist or cannot be accessed")
-
```

Comment:
we should still raise this exception if the path doesn't exist.

in the changes below, there's a check and error if the default path is used, but when the user provides `--signatures=/does/not/exist` which should still be handled here.

Reply:
Fixed